internal/proposals/logs/improvement-of-log-messages-usability.md

```diff
@@ -37,6 +37,7 @@ To unify the logs, which will make the debugging process and logs parsing much e
     - message language: English. To preserve unified number/date formatting, go with the `en-us` locale.
     - encoding: `UTF-8`.
 - **context** - structure of the contextual information, such as operation (for example: `starting workers`), handler/ resolver (for example: `ProvisionRuntime`), controller, resource-namespaced name (for example: `production/application1`), operation ID, instance ID, operation stage, and so on. Users must be able to filter the logs so all the info provided here must be a useful and unique minimal set for every operation. Users must be able to find the needed resource in some store so provide here a name instead of an ID if it's easier to use later.
+- **caller** - string with information from where the log was created eg.: `"pkg/components/client.go:45"`. It's helpful to quickly track the log.  
```

Comment:
```suggestion
- **caller** - string with information where the log was created, where it comes from. For example, `"pkg/components/client.go:45"`. It allows for quick log tracking.  
```
`It's helpful to quickly track the log.` might be read as "Quick log tracking helps.", and what you mean here is quite the opposite. You want to say "Such information helps with quick log tracking, makes quick log tracking easier.", right?

Reply:
Yes

Reply:
```suggestion
- **caller** - string with information where in the code the log was created. For example: `"pkg/components/client.go:45"`.
```